MiniTwit.Web.App/Controllers/ApiController.cs

```diff
@@ -34,17 +34,21 @@ IUserRepository userRepository
             _userRepository = userRepository;
         }
 
+        private void LogInfo (string message)
+        {
+            _logger.LogInformation($"Latest:{_latest}, {message}");
+        }
+
         //This function is defined, dont know if we need to use it.
         //The tests doesnt test for it.
-        private (bool, IActionResult) NotRequestFromSimulator()
+        private bool RequestFromSimulator()
```

Comment:
We will have to change this a bit soon, since people other than the simulator will need to usethe API, buts it's fine for now.